Resources/config/yui.yml

```diff
@@ -292,11 +296,45 @@ system:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/loginform.hbt
                 ez-dashboardview:
-                    requires: ['ez-templatebasedview', 'dashboardview-ez-template', 'ez-height-fit', 'event-resize']
+                    requires: ['ez-templatebasedview', 'ez-height-fit', 'dashboardview-ez-template']
                     path: %ez_platformui.public_dir%/js/views/ez-dashboardview.js
                 dashboardview-ez-template:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/dashboard.hbt
+                ez-dashboardblockview:
+                    requires: ['ez-templatebasedview', 'dashboardblockview-ez-template']
+                    path: %ez_platformui.public_dir%/js/views/dashboard/ez-dashboardblockview.js
+                dashboardblockview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard/dashboard-block.hbt
+                ez-allcontent-dashboardblockview:
+                    requires: ['ez-dashboardblockview', 'ez-allcontent-dashboardblockrowview', 'allcontentdashboardblockview-ez-template']
+                    path: %ez_platformui.public_dir%/js/views/dashboard/ez-allcontent-dashboardblockview.js
+                allcontentdashboardblockview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard/allcontent-dashboard-block.hbt
+                ez-allcontent-dashboardblockrowview:
+                    requires: ['ez-templatebasedview', 'event-outside', 'event-tap', 'allcontentdashboardblockrowview-ez-template']
+                    path: %ez_platformui.public_dir%/js/views/dashboard/ez-allcontent-dashboardblockrowview.js
+                allcontentdashboardblockrowview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard/allcontent-dashboard-block-row.hbt
+                ez-allcontent-dashboardblockserviceplugin:
+                    requires:
+                        - 'ez-viewservicebaseplugin'
+                        - 'ez-pluginregistry'
+                        - 'ez-contentmodel'
+                        - 'model-list'
+                        - 'promise'
+                    dependencyOf: ['ez-dashboardviewservice']
```

Comment:
we don't do that in PlatformUI since you can directly change the dependencies of `ez-dashboardviewservice`